docs/5-Snippets/Mailhog.md

```diff
@@ -0,0 +1,102 @@
+# Mailhog
```

Comment:
Just a comment - we have very common snippets for `elasticsearch`, `ssh tunneling`, `caddy proxy` - which we can add next to have a better body of content in this section to start with.